compliance/azure/subscription_access/azure_subscription_access.pt

```diff
@@ -1,10 +1,16 @@
 name "Azure Subscription Access"
 rs_pt_ver 20180301
 type "policy"
-short_description "Lists anyone who has been granted Owner or Contributor access to an Azure subscription. \n See the [README](https://github.com/rightscale/policy_templates/tree/master/compliance/azure/subscription_access) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
-long_description "Version: 1.3"
+short_description "Lists anyone who has been granted Owner or Contributor access to an Azure subscription. \n See the [README](https://github.com/flexera/policy_templates/tree/master/compliance/azure/subscription_access) and [docs.rightscale.com/policies](https://docs.rightscale.com/policies/) to learn more."
+long_description ""
 category "Compliance"
 severity "low"
+info(
+  version: "2.0",
+  provider: "Azure",
+  service: "Identity",
+  policy_set: "Azure Subscription Access"
```

Comment:
this policy template doesn't have a policy set

Reply:
Update.